src/components/UI/Card/Card.module.scss

```diff
@@ -0,0 +1,8 @@
+.card {
+  background-color: rgba(255, 255, 255, 0.5);
+  border-radius: 0.5rem;
+  box-shadow: 2px 2px 5px 0px #245f763b;
+  border: 1px solid #245f763b;
```

Comment:
the same about color, variable can be even moved to some file with common styles and imported from it

Reply:
Thank you! @import! I was struggling a lot when was trying to use newer syntax @use and did not understand why it did not work.